packages/driver/cypress/integration/commands/navigation_spec.js

```diff
@@ -1410,22 +1410,9 @@ describe('src/cy/commands/navigation', () => {
         cy.on('fail', (err) => {
           const { lastLog } = this
 
-          expect(err.message).to.equal(stripIndent`\
-          \`cy.visit()\` failed because you are attempting to visit a URL that is of a different origin.\n
-          You likely forgot to use \`cy.origin()\`:\n
-          \`cy.visit('http://localhost:3500/fixtures/generic.html')\`
-          \`<other commands targeting http://localhost:3500 go here>\`\n
-          \`cy.origin('http://localhost:3501', () => {\`
-          \`  cy.visit('http://localhost:3501/fixtures/generic.html')\`
-          \`  <other commands targeting http://localhost:3501 go here>\`
-          \`})\`\n
-          The new URL is considered a different origin because the following parts of the URL are different:\n
-            > port\n
-          You may only \`cy.visit()\` same-origin URLs within a single test.\n
-          The previous URL you visited was:\n
-            > 'http://localhost:3500'\n
-          You're attempting to visit this URL:\n
-            > 'http://localhost:3501'`)
+          expect(err.message).to.include('`cy.visit()` failed because you are attempting to visit a URL that is of a different origin.')
```

Comment:
Why where these changed?

Reply:
I reverted these back to how the test was prior to our mulit-domain change, but now that you mention it I could probably validate more. Let me check on it.